content/pages/playbook/design/visual.html

```diff
@@ -29,7 +29,7 @@
 
           <dl class="card">
             <dt>Headings: Roboto Slab, Source Sans Pro</dt>
-            <p>Content headings (headings 1 and 2 are reserved for the vets.gov header and breadcrumb navigation)</p>
+            <dt>Note that headings 1 and 2 are reserved for the vets.gov header and breadcrumb navigation</dt>
```

Comment:
@melwoodard I made an editorial change here, but LMK if I should leave as is or if you have a better suggestion.